src/services/azure/authConfig.js

```diff
@@ -0,0 +1,28 @@
+export const msalConfig = {
+    auth: {
+      clientId: "1308c1b6-585b-4b12-ba62-7cfd2790849f",
+      //authority:"https://login.microsoftonline.com/45155b2d-22e5-40e1-9ca6-cf423ef564be",
+      authority: "https://byo5.b2clogin.com/byo5.onmicrosoft.com/B2C_1_test1",
+      // eslint-disable-next-line
+      redirectUri: location.protocol+'//'+location.host+location.pathname,
+      //redirectUri: "http://localhost:3000/dashboard",
```

Comment:
@Darthcedeus Is the url specified in azure too, or can this be changed to anything here, like I have done above?

Reply:
@fifthsegment redirect url has to be specified in azure portal as well.

Reply:
oh!
okay, phir localhost he rehnay detay hayn